client/src/App.js

```diff
@@ -31,6 +43,46 @@ function App() {
         </Col>
       </Row>
     </Container>
+  ); */
+
+  return (
+    <Router>
+      <Container>
+        <Row>
+          <NavbarComponent cart={1}/>   
+          {/*cart prop receives the length of the items list in shopping cart */}
+        </Row>
+        <Row className='vh-100 justify-content-center align-items-center'>
+          <Switch>
+            <Route exact path="/">
+              <ProducutListPage/>
+            </Route>
+            <Route exact path="/cart">
+              <ShoppingCartPage/> 
+            </Route>
+            <Route exact path="/orders">
+              <OrdersPage/> 
+            </Route>
+            <Route exact path="/clients">
+              <ClientManagementPage/>
+            </Route>
+            <Route exact path="/clients/:id">
+              <ClientDetailsPage/>
+            </Route>
+            <Route exact path="/neworder">
+              <NewOrderPage/> {/* = ProductListPage in creating an order mode */}
```

Comment:
I'm pretty sure this page is useless. When the employee wants to create a user, he clicks on the create order button in the ClientDetailsPage and gets redirected to the ProductListPage, passing a prop "clientOverride" which contains the ID of the client the new order will be associated to.

ClientDetailsPage -> createOrder(clientID) -> ProductListPage(clientOverride: clientID) -> NewShoppingCart(clientOverride) -> AddProductToCart(productID, quantity)